docs/data/sdks/sdk-maintenance-and-support.md

```diff
@@ -0,0 +1,88 @@
+
+- **Maintenance (Phase 3)** - During the maintenance mode, Amplitude limits SDK releases to address critical bug fixes and security issues only. An SDK will not receive API updates for new or existing services, or be updated to support new regions. Maintenance mode has a default duration of 12 months, unless otherwise specified.
+
+- **End-of-Support (Phase 4)** - When an SDK reaches end-of support, it will no longer receive updates or releases. Previously published releases will continue to be available via public package managers and the code will remain on GitHub. The GitHub repository may be archived. Use of an SDK which has reached end-of-support is done at the user’s discretion. We recommend users upgrade to the new major version.
+
+## Dependency life-cycle
+
+Most Amplitude SDKs have underlying dependencies, such as language runtimes, operating systems, or third party libraries and frameworks. These dependencies are typically tied to the language community or the vendor who owns that particular component. Each community or vendor publishes their own end-of-support schedule for their product.
+
+The following terms are used to classify underlying third party dependencies:
+
+- **Operating System (OS):** Examples include MacOS 12.6, Android 13, Windows 11, etc.
+- **Language Runtime**: Examples include TypeScript 4.1.6, Swift 5.7, Java 8, Python 3.10.4, etc.
+- **Third party Library / Framework**: Examples include OpenSSL, OCLIF, React, etc.
+
+Our policy is to continue supporting SDK dependencies for at least 6 months after the community or vendor ends support for the dependency. This policy, however, could vary depending on the specific dependency.
+
+!!! note
+    Amplitude reserves the right to stop support for an underlying dependency without increasing the major SDK version
+
+## Communication methods
+
+Maintenance announcements are communicated in several ways:
+
+An email announcement is sent to affected accounts, announcing our plans to end support for the specific SDK version. The email will outline the path to end-of-support, specify the campaign timelines, and provide upgrade guidance.
+
+Amplitude SDK documentation, such as API reference documentation, user guides, SDK product marketing pages, and GitHub readme(s) are updated to indicate the campaign timeline and provide guidance on upgrading affected applications.
+
+An Amplitude blog post is published in the [Amplitude Community](https://community.amplitude.com/) that outlines the path to end-of-support, as well as reiterates the campaign timelines.
+
+Deprecation warnings are added to the SDKs, outlining the path to end-of-support and linking to the SDK documentation.
```

Comment:
"added to the SDKs" means "added to README.md" here?

Reply:
READMEs are called out above. This is in code, which we don't do today but probably should.